spec/request_handler/fieldsets_parser_spec.rb

```diff
@@ -3,21 +3,25 @@
 require 'spec_helper'
 describe RequestHandler::FieldsetsParser do
   let(:opts) do
-    Confstruct::Configuration.new do
+    build_docile(RequestHandler::Builder::OptionsBuilder, &block)
+  end
+
+  let(:block) do
+    proc do
       fieldsets do
         allowed do
-          posts Dry::Types['strict.string'].enum('awesome', 'samples')
-          photos Dry::Types['strict.string'].enum('foo', 'bar')
-          videos true
-          musicfiles false
+          resource :posts, Dry::Types['strict.string'].enum('awesome', 'samples')
```

Comment:
If possible, please switch to `Dry::Types::Strict::String` (in all occurences of this `.[]` syntax)

Reply:
Not sure but I don't think it's discouraged to use the `[]` in the latest version of dry-types...
```
dry-types does not define constants for default types. You can access the predefined types with [], e.g. Dry::Types["integer"] or generate a module with types using Dry.Types()
```

Do you think this would be better?
`Dry.Types()::String.default('red').enum('blue', 'green', 'red')`

Reply:
personally this `Dry::Types['strict.string']` looks nicer then `Dry.Types()::String`. but I guess here in the specs both is fine